app/helpers/application_helper.rb

```diff
@@ -516,40 +516,32 @@ def category_badge(category, opts = nil)
     CategoryBadge.html_for(category, opts).html_safe
   end
 
-  def self.all_connectors
-    @all_connectors = Dir.glob("plugins/*/app/views/connectors/**/*.html.erb")
-  end
-
-  PLUGIN_OUTLET_TEMPLATE_CACHE = Concurrent::Map.new
+  SERVER_PLUGIN_OUTLET_PLUGINS_PREFIX = Rails.env.test? ? "spec/fixtures/plugins/" : "plugins/"
```

Comment:
Is there any way we can include both paths when in test env? If we ignore the real plugins in the test env, it becomes impossible for plugins to write tests for their connectors.

Reply:
Good point 👍 Will work on a fix.